config/base.webpack.config.js

```diff
@@ -43,7 +43,7 @@ const webpackConfig = {
   },
 
   resolve: {
-    // modules: [resolve('./node_modules')], // uncomment when using linked package
+    ...(process.env.LINKED && { modules: [resolve('./node_modules')] }),
```

Comment:
Is this suppose to reference the linked package node_modules? Would it be better to resolve the sources node_modules? Like with the aliases bellow?

Reply:
It works, that's enough for me. :man_shrugging: